### ISSUE TITLE
install package source

### ISSUE THREAD

**Issue by jgrill** [irvingpop/supermarket-omnibus-cookbook#10]
Setting `default['supermarket_package']['package_source']` doesn't work.  The expected result is that the package would be downloaded and installed from the provided URL.  The actual result is that nothing is installed and...

```
    ===============================================================================
             Error executing action `run` on resource 'execute[supermarket-reconfigure]'
             ================================================================================

             Errno::ENOENT
             -------------
             No such file or directory - /opt/supermarket/bin/supermarket-ctl

             Resource Declaration:
             ---------------------
             # In /tmp/kitchen/cache/cookbooks/chef-server-ingredient/libraries/chef_server_ingredients_provider.rb

       76:         execute "#{new_resource.package_name}-reconfigure" do
       77:           command "#{ctl_command} reconfigure"
       78:         end
       79:       end
```

Related?  https://github.com/chef/chef/issues/3487

UPDATE: After reading the above issue, I wrote a quick-and-dirty recipe to simply call `rpm_package` with `source` pointing to a local, downloaded supermarket-1.11.0-alpha.0-1.rpm. It also failed to install!

I upgraded from chef-client 12.2.1 to 12.4.0 and tried it again and it worked.

**Comment by irvingpop** [comment]
hey @jgrill this was an issue in Chef client itself, which I fixed in 12.4.0:  https://github.com/chef/chef/pull/3503

**Comment by jgrill** [comment]
Thank you @irvingpop!
